CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Config option `window.gtk_theme_variant` to set GTK theme variant
 - Completions for `--class` and `-t` (short title)
 - Change the mouse cursor when hovering over the message bar and its close button
+- Text mouse cursor is shown when shift-selecting text
```

Comment:
This comment is a bit imprecise. It should state that the text mouse cursor is shown when holding shift while in mouse mode.